### ISSUE TITLE
Undead do not despawn if no player is in the zone to cause tick

### ISSUE THREAD

**Issue by Bizzle-Dapp** [DarkstarProject/darkstar#5882]
**_I have:_**
Initially noticed a Wight roaming Rolanberry fields in the middle of the daytime. Have since spotted Zombies in Pashlow Marshlands and a previous issue: https://github.com/DarkstarProject/darkstar/issues/2570 references a very similar issue. 

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:** 
30190328_2

**_Source Branch_** (master/stable) **:** 
Initially from whasf installer - since slightly editted

 Likely all
**_Additional Information_** (Steps to reproduce/Expected behavior) **:**

**Comment by KnowOne134** [comment]
Ive had people say similar. Ghoul in dunes as well as others here and there. I just tell then more mobs == more xp amd drops

**Comment by TeoTwawki** [comment]
yeah we're supposed to be checking and doing some thingies at 1st zone tick after a frozen (empty)  zone "wakes up" (somebody entered) and we currently are not.

check timers for NMs..check weather/elementals.. check undead...other things Teo can't think of right now.. we do all of this when zone is populated but only some of them when it goes empty and takes a nap and not right when it comes up from naptime.

**Comment by Bizzle-Dapp** [comment]
I agree, not a huge problem by any means, but thought best to mention it.
